### PR TITLE
Worked with Dapper and used CRUD 

### PR DESCRIPTION
**Pull request by karki011**
Finished with

- SELECT all products
- DELETE a product with a particular ProductId
- UPDATE a product's name with a particular ProductId
- INSERT a new product

Done using Dapper.

### CODE REVIEW

File: SqlIntro/IProductRepository.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+
+namespace SqlIntro
+{
+    public interface IProductRepository
+    {
+        void DeleteProduct(int id);
+        IEnumerable<Product> GetProducts();
+        void InsertProduct(Product prod);
+        void UpdateProduct(Product prod);
```
Comment (codytwinton) [inline SqlIntro/IProductRepository.cs]: Missing Inner and Left Join stubbed out methods here.

File: SqlIntro/Program.cs [hunk 2]
```diff
@@ -1,22 +1,44 @@
...
     {
         static void Main(string[] args)
         {
-            var connectionString = ""; //get connectionString format from connectionstrings.com and change to match your database
-            var repo = new ProductRepository(connectionString);
+            var connectionString = ConfigurationManager.ConnectionStrings["MySQLConnection"].ConnectionString;
+            var connection = new MySqlConnection(connectionString);
+            var repo = new DapperProductRepo(connection);
+
+            Product product = null;
```
Comment (codytwinton) [inline SqlIntro/Program.cs]: This value is never used, so you can probably remove it.

File: SqlIntro/Program.cs [hunk 3]
```diff
@@ -1,22 +1,44 @@
...
+            };
+            repo.InsertProduct(newProduct);
+
+            repo.UpdateProduct(new Product
+            {
+                Id =3,
+                Name = "Subash karki"
+            });
+
+            /*Console.WriteLine("Start.");
```
Comment (codytwinton) [inline SqlIntro/Program.cs]: Always start using or delete unused code.

File: SqlIntro/getAllProducts.cs [hunk 4]
```diff
@@ -0,0 +1,6 @@
+﻿namespace SqlIntro
+{
+    internal class getAllProducts
```
Comment (codytwinton) [inline SqlIntro/getAllProducts.cs]: Not sure what this is.

File: SqlIntro/IProductRepository.cs (hunk 1 above)
Comment (karki011) [reply SqlIntro/IProductRepository.cs]: pushed missing method

**Pull request closed without merging**